README.md

```diff
@@ -1,5 +1,5 @@
 
-[![CircleCI](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-Android/tree/dev.svg?style=svg)](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-Android/tree/dev)
+[![CircleCI](https://circleci.com/gh/forcedotcom/SalesforceMobileSDK-Android/tree/dev.svg?style=svg)](https://circleci.com/gh/forcedotcom/workflows/SalesforceMobileSDK-Android/tree/dev)
```

Comment:
@bhariharan Unfortunately this is the best I could do.  Clicking the badge now links to the workflows list of the dev branch but there is no way (at this time) to point it a specific workflow or job.  CircleCI is aware of the issue but hasn't fixed it yet.

Reply:
This is good.